contracts/SwapAndBridge.sol

```diff
@@ -103,8 +105,11 @@ abstract contract SwapAndBridgeBase is Lockable, MultiCaller {
         // user on this contract, a malicious actor could call transferFrom to steal the user's tokens.
         if (!allowedSelectors[bytes4(routerCalldata)]) revert InvalidFunctionSelector();
 
-        // Pull tokens from caller into this contract.
-        _swapToken.safeTransferFrom(msg.sender, address(this), swapTokenAmount);
+        // Pull tokens from caller into this contract. This contract may already have tokens if, for example,
+        // a user sent the contract tokens and subsequently called `swapAndBridge`, so only pull funds if
```

Comment:
I think this comment is a bit out of place now that we've added _pullTokens

Reply:
Removed https://github.com/across-protocol/contracts/pull/751/commits/7d67ab56d693289fd6d71933b4a5084503139a07